File: Selenium/Activities/Activity9.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class Activity9 {
    public static void main(String[] args) {
        WebDriverManager.chromedriver().setup();
        WebDriver driver = new ChromeDriver();
        Actions builder = new Actions(driver);
        driver.get("https://v1.training-support.net/selenium/ajax");
        System.out.println("Title of webpage : " + driver.getTitle());
        driver.findElement(By.xpath("//button[text()='Change Content']")).click();
        WebDriverWait wait=new WebDriverWait(driver,Duration.ofSeconds(60));
        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.tagName("h1")));
        System.out.println("message displaying in page : "+ driver.findElement(By.tagName("h1")).getText());
        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.tagName("h3")));
        System.out.println("message displaying in page : "+ driver.findElement(By.tagName("h3")).getText());
        driver.close();
    }
}

File: Selenium/Activities/ACtivity9.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver import ActionChains
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    actions = webdriver.ActionChains(driver)
    driver.get("https://v1.training-support.net/selenium/ajax")
    pageTitle=driver.title
    print(pageTitle)
    wait=WebDriverWait(driver,20)
    driver.find_element(By.XPATH,"//button[text()='Change Content']").click();
    wait.until(EC.visibility_of_element_located(By.TAG_NAME,"h1"))
    print("message displaying in page : ",driver.find_element(By.TAG_NAME,"h1").text)
    wait.until(EC.visibility_of_element_located(By.TAG_NAME,"h3"))
    print("message displaying in page : ",driver.find_element(By.TAG_NAME,"h3").text)
    driver.close()



